index.js

```diff
@@ -0,0 +1,32 @@
+// Toogle Mobile Menu javascript
+const linkPortfolio = document.querySelector('.port-link');
+const linkAbout = document.querySelector('.abt-link');
+const linkContact = document.querySelector('.contact-link');
+const menuBtn = document.querySelector('.menu-button');
+const btnCloseNav = document.querySelector('.btn-close-nav');
+const headerNavMobile = document.querySelector('.mobile-navigation');
+
+menuBtn.addEventListener('click', () => {
+  menuBtn.classList.add('hide');
+  headerNavMobile.classList.remove('hide');
+});
+
+btnCloseNav.addEventListener('click', () => {
+  menuBtn.classList.remove('hide');
+  headerNavMobile.classList.add('hide');
+});
+
+linkPortfolio.addEventListener('click', () => {
+  headerNavMobile.classList.add('hide');
+  menuBtn.classList.remove('hide');
+});
+
+linkAbout.addEventListener('click', () => {
+  headerNavMobile.classList.add('hide');
+  menuBtn.classList.remove('hide');
+});
+
+linkContact.addEventListener('click', () => {
+  headerNavMobile.classList.add('hide');
+  menuBtn.classList.remove('hide');
+});
```

Comment:
[OPTIONAL] Great job with the functionality 👍  But we can DRY them a little bit 👍  You can use the codes below:
```
// Toogle Mobile Menu javascript
const menuBtn = document.querySelector('.menu-button');
const btnCloseNav = document.querySelector('.btn-close-nav');
const headerNavMobile = document.querySelector('.mobile-navigation');
const navigationLinks = document.querySelector('.navigation-links');

menuBtn.addEventListener('click', () => {
  menuBtn.classList.add('hide');
  headerNavMobile.classList.remove('hide');
});

btnCloseNav.addEventListener('click', () => {
  menuBtn.classList.remove('hide');
  headerNavMobile.classList.add('hide');
});

navigationLinks.addEventListener('click', (event) => {
  if (event.target.matches('.port-link, .abt-link, .contact-link')) {
    headerNavMobile.classList.add('hide');
    menuBtn.classList.remove('hide');
  }
});
```

Reply:
Hi, @huseyinbiyik...
Thanks for the sugesstion...